service/db/dgclient_test.go

```diff
@@ -87,14 +89,58 @@ func TestCreateIndex(t *testing.T) {
 	client.DropSchema("testindex")
 }
 
-func TestGetSchema(t *testing.T) {
+func TestGetSchemaFromDB(t *testing.T) {
 	client := NewDGClient("127.0.0.1:9080")
 	defer client.Close()
-	smds, err := client.GetSchema()
+	smds, err := client.GetSchemaFromDB()
 	if err != nil {
-		log.Fatalf("failed to get schema %v", err)
+		log.Fatalf("failed to get schema db %v", err)
 	}
-	log.Infof("From test schema returned: [%v]\n", smds)
+	log.Infof("From TestGetSchemaFromDB returned: [%v]\n", smds)
+	assert.Nil(t, err)
+}
+
+func TestGetSchemaFromCache(t *testing.T) {
+	client := NewDGClient("127.0.0.1:9080")
+	defer client.Close()
+
+	//Creates an LRU cache of the given size
+	var err error
+	LruCache, err = lru.New(5)
```

Comment:
if the value `5` gets constant-ized as suggested in a comment on `server.go`, maybe use that here, as well

Reply:
Made the changes to add a constant for the cache size and set default to 10. :)